### ISSUE TITLE
BUG: Unexpected Dual Normalizing from Dataset Class and Alubmentations normalize

### ISSUE THREAD

**Issue by WendellAtHome** [4uiiurz1/pytorch-nested-unet#62]
There is a normalizing step in dataset.py line 71

`img = img.astype('float32') / 255`

Combined with transforms.Normalize() from alubmentations.augmentations 0.5.2,
where the source code (albumentations 0.5.2) is 

```
mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)

def normalize(img, mean, std, max_pixel_value=255.0):
    mean = np.array(mean, dtype=np.float32)
    mean *= max_pixel_value

    std = np.array(std, dtype=np.float32)
    std *= max_pixel_value

    denominator = np.reciprocal(std, dtype=np.float32)

    img = img.astype(np.float32)
    img -= mean
    img *= denominator
    return img
```

**The image is divided by 255 twice before input into the network!!!**
This tooks me some extra hours to debug during deploying in TensorRT.
Hope this issue helps next user.

**Comment by WendellAtHome** [comment]
If you try to convert the model you trained from the repo to ONNX, and write your own onnx inferencing. You are very likely to get very different output as the pytorch version, even if you divide the image with 255.0 twice following the repo implementation.
After a whole day of debuging, the issue is that after dual division, the values are smaller than expected, and the corresponding variance in batch normalization layer is even smaller (close to the the default epsilon 1e-5 in batch normalization). This causes unstability when converting to ONNX model.
